reference/forms/types/choice.rst

```diff
@@ -103,13 +103,12 @@ method::
             new Category('Cat3'),
             new Category('Cat4'),
         ],
-        // "name" is a property path, meaning Symfony will look for a public
-        // property or a public method like "getName()" to define the input
-        // string value that will be submitted by the form
+        // The value to be submitted by the form. `name` is a property path
+        // of the `Category` class
```

Comment:
I am not sure that removing the "meaning Symfony will look for a public property or a public method like "getName()" to define the input string value that will be submitted by the form" part is a good idea as it adds some useful information.

Reply:
Which part exactly do you mean?

* "Symfony will look for a public property or a public method like `getName()`": I figured that this is pretty common nowadays, and probably used in many places in the docs (but I didn't double check), so people will probably already know this concept.

* "to define the input string value that will be submitted by the form": If something is called `choice_value`, it's pretty clear that it is about, well, the choice's value ;-)

GitHub: I'm not sure if I'm supposed to press "Add single comment" or "Start a review" now ;-) I'm taking the "comment"...